[TYR][DOC]billing_plan_id parameter for users (#3396)
CanalTP/navitia: source/tyr/readme.md

###### POST Parameters

name            | description                                                                          | required | default                            |
----------------|--------------------------------------------------------------------------------------|----------|------------------------------------|
email           | adress email of the user                                                             | yep      |                                    |
login           | login of the user                                                                    | yep      |                                    |

> type            | type of the user between: [with_free_instances, without_free_instances, super_user]  | nope     | with_free_instances                |

Reviewer: ```suggestion
type            | type of the user, one of the following values: [with_free_instances, without_free_instances, super_user]  | nope     | with_free_instances                |
```

> end_point_id    | the id of the endpoint for this user (in most case the default value is good enough  | nope     | the default end_point (navitia.io) |

Reviewer: ```suggestion
end_point_id    | the id of the endpoint for this user (in most cases the default value is good enough  | nope     | the default end_point (navitia.io) |
```

shape           | the shape (geojson) to be used for autocomplete for this user                        | nope     | null                               |

> billing_plan_id | the id of the billing_plan for this user                                             | nope     | Will use the default billing plan  |

Reviewer 2: ```suggestion
billing_plan_id | the id of the billing_plan for this user                                             | nope     | the default billing plan  |
```
but what's default ?
If known it can be put into parenthesis
